lib/index.js

```diff
@@ -93,14 +91,20 @@ function increaseDifficulty(overlord, world){
 }
 
 function checkLevel(){
-  var level = document.getElementById('level'); 
+  var level = document.getElementById('level');
   if(trayCollision.total >= 200 && trayCollision.total < 400 ){
     level.innerHTML = "<h3>Level 2</h3>";
   } else if (trayCollision.total >= 400 && trayCollision.total < 600 ){
     level.innerHTML = "<h3>Level 3</h3>";
   }
 }
 
+function checkGameEnders(){
+  if(trayCollision.total < 0 && gameRunning || trayCollision.total >= 600 && gameRunning || !gameRunning){
```

Comment:
I think this is better as one if statement since the outcome is the same for each of these statements/conditionals. They all return endGame(); if met. Thanks.